### DISCUSSION TITLE
Terminal & Font

### DISCUSSION THREAD

**Discussion by sillypnk** [NvChad/NvChad#896] [Q&A]
What is your terminal and font used in screenshots?

**Comment by siduck** [comment]
st terminal with jetbrainsmono nerd font. check my st repo

**Reply by sillypnk** [reply]
Thanks, but st is for linux and mac only right? Btw thanks for the font

**Reply by siduck** [reply]
st is only for linux, i suggest using alacritty or kitty for windows/mac

**Reply by sillypnk** [reply]
> st is only for linux, i suggest using alacritty or kitty for windows/mac

kitty doesn't support windows 👀 But will st programs run if i install wsl?

**Reply by siduck** [reply]
if your wsl supports xorg stuff then yes st might work
